library/xml-model/src/test/java/org/cru/godtools/xml/model/tract/TractPageTest.kt

```diff
@@ -43,6 +43,8 @@ class TractPageTest {
         assertThat(page.cards, hasSize(2))
         assertEquals("Card 1", page.cards[0].label!!.text)
         assertEquals("Card 2", page.cards[1].label!!.text)
+        assertEquals(page.cards[0].backgroundColor, page.cardBackgroundColor)
+        assertEquals(page.cards[1].backgroundColor, page.cardBackgroundColor)
```

Comment:
this is basically testing the card parsing (already tested in `CardTest`), as well as the page card background color fallback (also already tested in `CardTest`).

I'd rather see that the parsing logic correctly reads page.cardBackgroundColor from the XML

Reply:
So should I put that inside of a separate test or could I add the assertions inside of something like `verifyParseCards()`  where we don't define a ```cardBackgroundColor``` so it should fall back to ```manifest.cardBackgroundColor```?

Reply:
I would define a separate test method to test the page cardBackgroundColor fallback behavior